libs/ledger-live-common/src/featureFlags/defaultFeatures.ts

```diff
@@ -399,36 +398,13 @@ export const DEFAULT_FEATURES: Features = {
   },
   ptxCard: DEFAULT_FEATURE,
 
-  ptxSwapLiveAppDemoZero: {
-    enabled: false,
-    params: {
-      manifest_id: "swap-live-app-demo-0",
-    },
-  },
-
-  ptxSwapLiveAppDemoOne: {
-    enabled: false,
-    params: {
-      manifest_id: "swap-live-app-demo-1",
-    },
-  },
-
   ptxSwapLiveAppDemoThree: {
```

Comment:
We should remove the demo3 in the name and only use ptxSwapLiveApp

Reply:
replaced